common/religion/doctrines/wc_doctrines_magic.txt

```diff
@@ -0,0 +1,1500 @@
+﻿doctrine_light_magic = {
```

Comment:
I noticed a disturbing design thing. If you're to pick a syncretism tenet, you have to change magic laws increasing piety cost hightly.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/107852099-357c0700-6e28-11eb-8f4f-69c9d61ad0db.png)

</details>

Reply:
Made it cheaper in c824f1f7 , check it, please